.github/workflows/ci.yml

```diff
@@ -280,12 +297,24 @@ jobs:
             target
           key: coverage-cargo-${{ hashFiles('**/Cargo.toml') }}
         continue-on-error: true
-      - name: install cargo-llvm-cov
+      - if: matrix.os.name == 'windows'
+        name: install cargo-llvm-cov (windows)
+        shell: powershell
         run: |
-          wget https://github.com/taiki-e/cargo-llvm-cov/releases/download/v${CARGO_LLVM_COV_VERSION}/cargo-llvm-cov-x86_64-unknown-linux-gnu.tar.gz -qO- | tar -xzvf -
+          Invoke-WebRequest https://github.com/taiki-e/cargo-llvm-cov/releases/download/v${Env:CARGO_LLVM_COV_VERSION}/cargo-llvm-cov-${Env:CARGO_LLVM_COV_ARCH}.zip -OutFile cargo-llvm-cov.zip
+          Expand-Archive cargo-llvm-cov.zip
+          mv ./cargo-llvm-cov/cargo-llvm-cov.exe ~/.cargo/bin
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+          profile: minimal
+          components: llvm-tools-preview
+      - if: matrix.os.name != 'windows'
+        name: install cargo-llvm-cov (unix)
+        run: |
+          wget https://github.com/taiki-e/cargo-llvm-cov/releases/download/v${CARGO_LLVM_COV_VERSION}/cargo-llvm-cov-${CARGO_LLVM_COV_ARCH}.tar.gz -qO- | tar -xzvf -
           mv cargo-llvm-cov ~/.cargo/bin
```

Comment:
Btw, cargo-llvm-cov has both .tar.gz and .zip distribution on windows, so you can unify the installation script on both windows and unix (with `shell: bash`). For example:

```
host=$(rustc -Vv | grep host | sed 's/host: //')
curl -fsSL --retry 3 https://github.com/taiki-e/cargo-llvm-cov/releases/download/v${CARGO_LLVM_COV_VERSION}/cargo-llvm-cov-"$host".tar.gz | tar xzf - -C ~/.cargo/bin
```